scripts/setup.sh

```diff
@@ -45,7 +52,8 @@ DYNAMODB_ARN=$(aws --endpoint-url=http://localhost:4566 dynamodb create-table \
 
 echo "Created table, arn: ${DYNAMODB_ARN}"
 
-
+# media-downloader dependencies
+brew install ffmpeg phantomjs
```

Comment:
what about installing yt-dlp itself?

Reply:
Good point. I've updated the pr description to address this, and now removed this brew command from setup.sh.

The way whisper works at the moment is that the worker runs a docker container with whisper installed rather than actually executing whisper directly. I think we could do the same thing for media-download. 

At the moment there's no guidance on installing yt-dlp in this PR, pending the next PR with various dockerfiles and other infrastructure bits in it. I could add a Pipfile (pipenv is my fav new python thing since you showed it to me) but given that it will just be for a short while before we swap out running yt-dlp directly for docker maybe it's fine as is

Reply:
Ok, happy to wait for yt-dlp on docker